### PR TITLE
IGAPP-1234: Multipoi representation 2

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

This is the new Overall multipoi PR :) 
Review starting the commit `IGAPP-1234: rename PoiFeature`
@LeandraH I already addressed your requested changes. :)

EDIT: most of the changes are renaming PoiFeature to MapFeature, so this should go down pretty smoothly.

### CODE REVIEW

**Review by steffenkleinle — APPROVED** [review]
Things I realized:
1. The refreshing described in the web PR still happens to me :/ (https://github.com/digitalfabrik/integreat-app/pull/953#pullrequestreview-1510788836) (web). However, since I think this is not explicitly related to your PR, we can also create an issue for it or ignore it
2. The separation line here is somehow really thick:
image (web)
3. Seems like it actually fixes https://issues.tuerantuer.org/browse/IGAPP-1360, good job!
Tested on android and firefox

File: native/ios/Podfile.lock [hunk 1]
```diff
@@ -893,6 +893,6 @@ SPEC CHECKSUMS:
   Yoga: 39310a10944fc864a7550700de349183450f8aaa
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
-PODFILE CHECKSUM: e8b5741aa4f3d7621c104abd7aec564babbf2664
+PODFILE CHECKSUM: c273128d3223ff1b4b77fc982f00e5cdfdc945e2
```
[the remark this answers is not in the document]
Comment (sarahsporck) [reply native/ios/Podfile.lock:896]: Not sure. Maybe someone forgot to pod install?

File: web/src/routes/PoisPage.tsx [hunk 2]
```diff
@@ -55,16 +55,19 @@ const PoisPage = ({ cityCode, languageCode, city, pathname }: CityRouteProps): R
...
   useEffect(() => {
     getUserLocation().then(userLocation =>
       userLocation.status === 'ready' ? setUserLocation(userLocation.coordinates) : null
     )
   }, [])
 
+  useEffect(
+    () => (city ? setMapViewport(moveViewToBBox(city.boundingBox!, defaultMercatorViewportConfig)) : undefined),
+    [city]
+  )
```
[the remark this answers is not in the document]
Comment (steffenkleinle) [reply web/src/routes/PoisPage.tsx]: So according to the cms team bounding box should always be set: https://chat.tuerantuer.org/digitalfabrik/pl/1o1rgw9fi3ny8qtqa1q7by571y
So we could actually adjust the types in the api-client/model.

[the remark this answers is not in the document]
Comment (sarahsporck) [reply web/src/routes/PoisPage.tsx]: Nice. then I'll adjust the type in this pr

**Review by LeandraH — APPROVED** [review]
Tested in Firefox and emulated iOS, very nice! Refreshing the page also works. Didn't check the code again.

**Pull request merged**